.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Release eda-server
+
+env:
+  LC_ALL: "C.UTF-8" # prevent ERROR: Ansible could not initialize the preferred locale: unsupported locale setting
+
+on:
+  workflow_dispatch:
+
+jobs:
+  stage:
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    permissions:
+      packages: write
+      contents: write
+    steps:
+      - name: Checkout eda-server
+        uses: actions/checkout@v3
+
+      - name: Get python version from Makefile
+        run: echo py_version=`make PYTHON_VERSION` >> $GITHUB_ENV
```

Comment:
This line would not going to fail? I don't see any PYTHON_VERSION in the makefile.

Reply:
I am still trying to find a way to test this.

Reply:
I added this make target